src/components/tests/ColumnChart.spec.tsx

```diff
@@ -128,4 +128,38 @@ describe("ColumnChart", () => {
         expect(wrapper.find(AfmColumnChart).prop("afm")).toEqual(expectedAfm);
         expect(wrapper.find(AfmColumnChart).prop("resultSpec")).toEqual(expectedResultSpec);
     });
+
+    describe("Stacking", () => {
+        const config = { stackMeasures: true, stackMeasuresToPercent: true };
+
+        it("should reset stackMeasuresToPercent in case of one measure", () => {
+            const wrapper = shallow(
```

Comment:
you can create some component factory parametrized only by Measure and use it in both test cases

Reply:
updated